models/core/transfers.sql

```diff
@@ -0,0 +1,49 @@
+{{ 
+    config(
+        materialized='incremental', 
+        unique_key= "CONCAT_WS('-', 'log_id', 'block_id', 'tx_hash', coalesce('from_address',''), coalesce('to_address',''), coalesce('contract_address',''), coalesce('raw_amount',-1))",
```

Comment:
i think only `log_id` will be needed here as unique key

Reply:
Resolved with commit: https://github.com/MetricsDAO/harmony_dbt/pull/19/commits/ff697e761495024a501d7b4a077ab8dcb0b215c2